examples/Flex.main.jsx

```diff
@@ -7,7 +7,7 @@ import { Flex } from '@itwin/itwinui-react';
 
 export default () => {
   return (
-    <Flex>
+    <Flex gap='l'>
```

Comment:
I think we should revert this specific change. The default `gap` is valuable to show, so that people have an idea of what it looks like by default.

Reply:
I have made this change.